device/mellanox/x86_64-mlnx_msn4700-r0/sensors.conf

```diff
@@ -0,0 +1,116 @@
+################################################################################
```

Comment:
need to clarify to be on the same size as this is not the same system.

Reply:
revised according to the bring up test result.